packages/xo-server/src/xo-mixins/backups-ng/index.js

```diff
@@ -639,7 +640,7 @@ export default class BackupNg {
         if (timeout !== 0) {
           const source = CancelToken.source([cancelToken])
           cancelToken = source.token
-          setTimeout(source.cancel, timeout)
```

Comment:
Are there other occurrences we should fix?

Reply:
For this PR yes.